@here/harp-mapview/lib/geometry/TileGeometryLoader.ts

```diff
@@ -167,6 +169,13 @@ export class SimpleTileGeometryLoader implements TileGeometryLoader {
         return this.m_isFinished;
     }
 
+    get isLoading(): boolean {
```

Comment:
Remark for refactoring later (i.e. proper TileLoadQueue)
- `isLoading` is confusing `geometryCreationPending` would be more clear
- We should have a proper state enum in the Tile like:
```
enum TileState {
    Loading, // request was send to the server
    Decoding, // decoding in the worker
    GeometryCreation, // THREE.js object creation
    Finished,
    Failed
}
```
Right now it is really hard to get the tile state b/c there are so many components that need to be checked (`TileLoader`, `TileGeometryLoader`, ...)

Reply:
Ok done.